scripts/bench_python_cpp_algos.py

```diff
@@ -29,28 +31,38 @@ def check_schema(graph: Graph, property_name):
     assert node_schema.names[new_property_id] == property_name
 
 
-def create_empty_statistics():
+def create_empty_statistics(args):
+    now = datetime.now(pytz.timezone("US/Central"))
     data = {
-        "btimestamp": 0,
-        "db": "analytics",
-        "etimestamp": 0,
+        "graph": args.graph,
+        "threads": args.threads,
+        "thread-spin": args.thread_spin,
+        "source-nodes": args.source_nodes,
+        "trials": args.trials,
+        "num-sources": args.num_sources,
         "import_time": 0,
         "num_partitions": 0,
         "routines": {},
-        "datetime": "",
+        "duration": time.time(),
```

Comment:
It is a bit unusual to set duration to be current time.  A bit of a hack to carry around beginning of execution.